jas/common.go

```diff
@@ -273,8 +273,27 @@ func convertToFilesExcludePatterns(excludePatterns []string) []string {
 	return patterns
 }
 
-func GetAnalyzerManagerXscEnvVars(msi string, technologies ...techutils.Technology) map[string]string {
+func CheckForSecretValidation(xrayManager *xray.XrayServicesManager, xrayVersion string, validateSecrets bool) bool {
+	dynamicTokenVersionMismatchErr := goclientutils.ValidateMinimumVersion(goclientutils.Xray, xrayVersion, jasutils.DynamicTokenValidationMinXrayVersion)
+	if dynamicTokenVersionMismatchErr != nil {
+		if validateSecrets {
+			log.Warn("Token validation (--validate-secrets flag) is not supported in your xray version")
+		}
```

Comment:
```suggestion
		if validateSecrets {
			log.Info("Token validation (--validate-secrets flag) is not supported in your xray version")
		}
```
add what version and what is the minimum in the log. I think you can print the error and the info is in it just wrap it with fmt and the extra info

Reply:
ok